<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>231</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Tx &amp; Rx</string>
    </property>
    <widget class="QSpinBox" name="sp_Tx">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>56</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Tx ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>56</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Rx ID:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_Rx">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>56</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_send">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>80</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>371</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Connect to Server</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>311</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_Mapdata">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>251</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect Server</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Setup</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>590</y>
        <width>281</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Node Setting</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>281</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Ground Station &amp; Nodes</string>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>241</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ground Station Coordinates:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Network</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Latitude:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>111</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Longtiude:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="pb_NodesCount">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>181</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Nodes Count:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sp_Lat">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>111</width>
         <height>33</height>
        </rect>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="value">
        <double>35.715297999999997</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sp_Long">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>111</width>
         <height>33</height>
        </rect>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
       <property name="value">
        <double>51.404342999999997</double>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_mapGrdView">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>121</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>MapGridView</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Grid Setting</string>
      </property>
      <widget class="QPushButton" name="pb_Apply">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>100</width>
         <height>33</height>
        </rect>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>241</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Selected Zone Size(km*km):</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="sp_zoneSize">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="sp_pixelLength">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>101</width>
         <height>28</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>181</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>Cells Per Side:</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Node Setting</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>461</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
